README.md

```diff
@@ -0,0 +1,54 @@
+# Javascript Weather-App
+
+> This is part of the Weather-App Project in [The Odin Project's](https://www.theodinproject.com/courses/javascript/lessons/weather-app?ref=lnav) Javascript Curriculum.
+
+![screenshot](./screenshot.PNG)
+
+View the app live [here](https://raw.githack.com/somoye123/Weather-app/feature/dist/index.html)
+
+## Built With
+
+- Javascript, HTML and CSS
+- VSCode, Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository and there you go! ;-)
+
+### Prerequisites
+
+- Get a browser like Chrome and Firefox in their most recent versions
+
+### Setup
+
+- git clone https://github.com/somoye123/Weather-app.git
+- Weather-app
```

Comment:
What happens in this step? I guess you should have a `cd` there.

Reply:
thanks
added the cd